src/sentry/api/serializers/models/group.py

```diff
@@ -422,7 +422,9 @@ def _get_status(self, attrs, obj):
                         "ignoreUntil": snooze.until,
                         "ignoreUserCount": (
                             snooze.user_count - (attrs["user_count"] - snooze.state["users_seen"])
-                            if snooze.user_count and not snooze.user_window
+                            if snooze.user_count
+                            and not snooze.user_window
+                            and not self._collapse("stats")
                             else snooze.user_count
```

Comment:
I'm not sure it's correct here to use `snooze.user_count` when collapse is used. The logic previously was "If either absolute user count or no user count is set, use `snooze.user_count`, otherwise use the dynamic value". This is because `snooze.user_count` is only useful when it's either `None` or over the lifetime of the snooze. If we always use `snooze.user_count` when we collapse, then the value of `ignoreUserCount` might be wrong.

It might be better to determine whether we use this value at all when we're collapsing, and maybe exclude it as well?

Reply:
Actually... did I get this backwards? Now that I'm reading this again, it looks like we only use the dynamic counts when there's not a dynamic time window. Now I'm not sure I understand the original logic at all

Reply:
I guess this is fine... in some cases we return the remaining delta, and others we return the total limit. I don't really understand why, maybe the FE is using them in tricky ways? I think ideally we wouldn't have these values conflated in the same key.